drake/systems/plants/@RigidBodyManipulator/signedDistances.m

```diff
@@ -0,0 +1,25 @@
+function [phi,n,x,body_x, body_idx] = signedDistances(obj,kinsol,points,use_margins)
```

Comment:
I see what you mean -- so something more like "collisionDetectFromPoints" would make sense?